portfolio/src/main/webapp/homepage.html

```diff
@@ -6,16 +6,17 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getComments()">
+  <body onload="checkStatus()">
     <div id="content">
       <h1>Leo's Portfolio</h1>
       <p>Click here to get a random greeting:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-      <p>click here to get a personalized greeting:</p>
+      <p>Enter your name here to get a personalized greeting:</p>
       <div id="special-greeting-container"></div><br>
       <input id="name-input" name="name-input" type="text" placeholder="What's your name?" onchange="addSpecialGreeting()">
-	  <form action="/data" method="POST">
+      <div id="commentDisplay">
+      <form action="/data" method="POST">
```

Comment:
fix the indent. The `<form>` component and child elements should have an additional 2 spaces.